service/vc/validator_test.go

```diff
@@ -89,33 +85,33 @@ func TestValidate(t *testing.T) { //nolint:maintidx // cannot improve.
 		"1": {
 			keys:     [][]byte{k1_1},
 			values:   [][]byte{[]byte("value1.1.1")},
-			versions: [][]byte{v2},
+			versions: []uint64{2},
 		},
 		"2": {
 			keys:     [][]byte{k2_1},
 			values:   [][]byte{[]byte("value1.1.1")},
-			versions: [][]byte{v2},
+			versions: []uint64{2},
 		},
 	}
 	tx2NonBlindWrites := namespaceToWrites{
 		"1": {
 			keys:     [][]byte{k1_5},
 			values:   [][]byte{[]byte("value1.5.1")},
-			versions: [][]byte{v0},
+			versions: []uint64{0},
 		},
 	}
 	tx3NonBlindWrites := namespaceToWrites{
 		"2": {
 			keys:     [][]byte{k2_2},
 			values:   [][]byte{[]byte("value2.2.1")},
-			versions: [][]byte{v2},
+			versions: []uint64{2},
 		},
 	}
 	tx3BlindWrites := namespaceToWrites{
 		"1": {
 			keys:     [][]byte{k1_6},
 			values:   [][]byte{[]byte("value1.6")},
-			versions: [][]byte{nil},
+			versions: []uint64{0},
```

Comment:
For future reference, we can add a comment that the version would not be included in the transaction for the blind-write and the vcservice would fetch the version from disk irrespective of what we pass here. Anyway, for this test, it does not matter what the version is. I believe this will help us in the future.